icu4c/source/i18n/measunit_impl.h

```diff
@@ -32,6 +32,8 @@ struct SingleUnitImpl : public UMemory {
     /** Transform this SingleUnitImpl into a MeasureUnit, simplifying if possible. */
     MeasureUnit build(UErrorCode& status) const;
 
+    const char *getIdentifier() const;
```

Comment:
Optional: more documentation please

Reply:
It would be useful to say that the returned identifier points to static memory and does not need to be cleaned up.

Reply:
getIdentifier was misleading because this won't return "square-kilometer", even though that would be the identifier for a MeasureUnit containing only this SingleUnit. I've renamed it to "getSimpleUnitID" as inspired by terminology from
https://unicode-org.github.io/cldr/ldml/tr35-general.html#Unit_Identifiers
- and I've improved the documentation.